catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/TemplateTransformer.java

```diff
@@ -218,19 +214,19 @@ private static String retrieveOwnerIfPresent(Metacard inputMetacard) {
    * response
    */
   private static Map<String, List<Serializable>> retrieveSecurityIfPresent(Metacard inputMetacard) {
-    List<Serializable> accessIndividuals = new ArrayList<>();
-    List<Serializable> accessGroups = new ArrayList<>();
-
-    if (inputMetacard.getAttribute(SecurityAttributes.ACCESS_INDIVIDUALS) != null) {
-      accessIndividuals.addAll(
-          inputMetacard.getAttribute(SecurityAttributes.ACCESS_INDIVIDUALS).getValues());
-    }
+    return ImmutableMap.of(
+        Security.ACCESS_INDIVIDUALS,
+        getValues(inputMetacard, SecurityAttributes.ACCESS_INDIVIDUALS),
+        Security.ACCESS_GROUPS,
+        getValues(inputMetacard, SecurityAttributes.ACCESS_GROUPS),
+        Security.ACCESS_ADMINISTRATORS,
+        getValues(inputMetacard, SecurityAttributes.ACCESS_ADMINISTRATORS));
+  }
 
-    if (inputMetacard.getAttribute(SecurityAttributes.ACCESS_GROUPS) != null) {
-      accessGroups.addAll(inputMetacard.getAttribute(SecurityAttributes.ACCESS_GROUPS).getValues());
+  private static List<Serializable> getValues(Metacard inputMetacard, String fieldName) {
+    if (inputMetacard.getAttribute(fieldName) != null) {
```

Comment:
Another option:
```return inputMetacard.getAttribute(fieldName) == null ? Collections.emptyList() : inputMetacard.getAttribute(fieldName).getValues()```

Reply:
stylistically, I like to avoid the ternary operator unless I'm using to provide a default value for assignment